src/ray/common/ray_config_def.h

```diff
@@ -242,6 +242,13 @@ RAY_CONFIG(uint32_t, worker_max_resource_analysis_iteration, 128);
 /// NOTE(swang): Linux only.
 RAY_CONFIG(int, worker_oom_score_adjustment, 1000)
 
+/// Sets workers' nice value on posix systems, so that the OS prioritizes CPU for other
+/// processes over worker. This makes CPU available to GCS, Raylet and user processes
+/// even when workers are busy.
+/// Valid value is [0, 19] (negative values require sudo permissions).
+/// NOTE: Linux, Unix and MacOS only.
+RAY_CONFIG(int, worker_niceness, 15)
```

Comment:
Is the default niceness different from platform & OS?

Reply:
and I wonder in this case, if we should just set low niceness on raylet instead of increasing worker niceness (e..g, set the lowest possible niceness on raylet)?

Reply:
since it means worker process can get less CPU than other processes like e.g., log moniitor

Reply:
Lowering niceness on Raylet requires `sudo`. Also, having processes like reporter agent, dashboard with lower niceness is desirable because they are required for debugging. IIUC log monitor is not actually related to log persistence, so if it turns out to consume huge amount of CPU we can optimize that and maybe give it a higher niceness.